README.md

```diff
@@ -98,6 +98,14 @@ generated docs will apppear in `docs/index.html`. Then you can point
 your browser to the absolute path of that file to review your local
 version of the docs.
 
+## Contributing to this repo
+
+When you try to commit your changes, a pre-commit hook with run. It will:
+1. Check linting
+2. Run tests
+
+If your tests fail, it is because your hook is looking for specific keys in your environment. In order to commit your changes and test on Github, you will need to pass `--no-verify` after your commit. 
```

Comment:
@BennyKitchell Are there env variables I'm supposed to set up somewhere? Like a lob-api key? If this is the case, I'd rather update the Readme to give that information to the user instead.

Reply:
Our contract tests (under the `tests/` directory) look for the `LOB_API_TEST_TOKEN` and `LOB_API_LIVE_TOKEN` env vars. I keep mine in `.zshrc` and run contract tests with commands like `npm run singleTest addresses` or `npm run test`. As far as I can recall, the contract tests are the only places that pull your local env vars.